### PR TITLE
Bug fix for https://bugs.launchpad.net/or/+bug/2091895 Train disappears from train list window

### PR DESCRIPTION
**Pull request by Csantucci**
See https://www.elvastower.com/forums/index.php?/topic/38454-changing-train-driven-by-the-player/

### CODE REVIEW

File: Source/Orts.Simulation/Simulation/Simulator.cs [hunk 1]
```diff
@@ -1955,14 +1955,14 @@ private void StartSwitchPlayerTrain()
...
                         }
-                        if (playerTrain.TrainType == Train.TRAINTYPE.AI_PLAYERDRIVEN || !playerTrain.Autopilot)
+                        if (playerTrain.TrainType == Train.TRAINTYPE.AI_PLAYERDRIVEN || TimetableMode && !playerTrain.Autopilot)
                         {
                             // it must be autopiloted first
                             playerTrain.SwitchToAutopilotControl();
                         }
                         // and now switch!
                         playerTrain.TrainType = Train.TRAINTYPE.AI;
-                        playerTrain.Autopilot = false;
```
Comment (cesarBLG) [inline Source/Orts.Simulation/Simulation/Simulator.cs:1965]: Is removing this intentional? The line is present in ORNYMG

Comment (Csantucci) [reply Source/Orts.Simulation/Simulation/Simulator.cs:1965]: Correcting an error and inserting another error.
Thanks!

**Review by cesarBLG — APPROVED** [review]
Thanks for the fix! Looks correct to me.

**Comment by Csantucci** [comment]
> Thanks for the fix! Looks correct to me.

You have suggested it, thanks!

**Pull request merged**